test/unit/util/pricing.spec.js

```diff
@@ -86,3 +87,28 @@ test('gets the price with custom duration config', (t) => {
     t.deepEqual(getDurationPrice(input), expected)
   })
 })
+
+test('computes price after fees', (t) => {
```

Comment:
Maybe `applies takerFeesPercent and ownerFeesPercent to ownerPrice`, more explicit.

Reply:
Thanks